niworkflows/interfaces/ants.py

```diff
@@ -516,3 +534,16 @@ def _list_outputs(self):
                     '%d', '*'))
             )
         return outputs
+
+
+def _copy_header(header_file, in_file, set_dtype=True):
+    """Copy header from input image to an output image."""
+    import nibabel as nb
+    in_img = nb.load(header_file)
+    out_img = nb.load(in_file, mmap=False)
+    new_img = out_img.__class__(out_img.get_fdata(), in_img.affine,
```

Comment:
@effigies - shouldn't this be using the `out_img.dataobj` instead?

Reply:
I would. It shouldn't change the values, just skip a cast or two.